addons/l10n_lt/data/account_account_type_data.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <!-- Additional Account Types for Lithuanian Accounting -->
+    <record id="account_account_type_intangible_assets" model="account.account.type">
```

Comment:
Can you explain us why do you need all these extra account types for Lithuanian accounting ?

Reply:
I will discuss this with our Finance Manager if those additional account types are really needed or standard ones are good enough. It does look like this list is a bit exhaustive.

Reply:
Merged most of account types and replaced with standard ones. But had to keep few ones, because there were no equivalent in standard types.

Reply:
can't they be replaced with current assets ?
cc: @jco-odoo

Reply:
@rgo-odoo after some debate with our Financial Manager, we managed to merge all account types with existing ones. They do not fit precisely, but I think it will be OK.